### ISSUE TITLE
Is it safe to use SECRET_KEY as the salt?

### ISSUE THREAD

**Issue by fjsj** [nshafer/django-hashid-field#1]
Looks like it's possible to discover the salt by looking at hashids: http://carnage.github.io/2015/08/cryptanalysis-of-hashids
This blog post is linked at hashids.org website: http://hashids.org/#decoding

Therefore, maybe this lib shouldn't use `SECRET_KEY` by default as the salt.
IMO there should be a custom Django setting for the global hashid salt.

**Comment by nshafer** [comment]
Thanks, I should have looked into that before I published. I have added a global setting and pushed a new version out. This of course will change all existing Hashids if just dropped in place, and no `salt` was specified on each field, so I upped it to version 1.0.0 since it's a breaking API change when used in its default state. Very regrettable, but hopefully it won't affect a lot of people. I have no idea how many are using this module since PyPI doesn't have counters anymore (not that they worked before with mirrors inflating them.) But at least you caught it somewhat early, as the module has only existed for a couple months. Thanks!
